Description: Revert "Fix treeview background, fixes #4855"
Origin: upstream, https://github.com/sugarlabs/sugar-toolkit-gtk3/commit/00ebb38
Author: Martin Abente Lahaye <tch@sugarlabs.org>
Forwarded: yes
Last-Update: 2015-07-01

--- a/gtk3/theme/gtk-widgets.css.em
+++ b/gtk3/theme/gtk-widgets.css.em
@@ -1,10 +1,5 @@
 ${
 import math
-from gi.repository import Gtk
-
-gtk_major = int(Gtk.get_major_version())
-gtk_minor = int(Gtk.get_minor_version())
-treeview_pseudo_element = gtk_major >= 3 and gtk_minor > 8
 
 def my_floor(num):
     return int(math.floor(num))
@@ -226,14 +221,10 @@
     border-width: 0px;
 }
 
-$[if treeview_pseudo_element] GtkTreeView row:even
-$[else] GtkTreeView row:nth-child(even)
-$[end if] {
+GtkTreeView row:nth-child(even) {
     background-color: @row_even;
 }
-$[if treeview_pseudo_element] GtkTreeView row:odd
-$[else] GtkTreeView row:nth-child(odd)
-$[end if] {
+GtkTreeView row:nth-child(odd) {
     background-color: @row_odd;
 }
 
