--- sugar-artwork-0.79.0.orig/cursor/sugar/Makefile.am
+++ sugar-artwork-0.79.0/cursor/sugar/Makefile.am
@@ -31,7 +31,7 @@
 install-data-local:
 	$(mkinstalldirs) $(DESTDIR)$(datadir)/icons/sugar/cursors/
 	for i in `cd $(THEME_DIR) && echo *` ; do					\
-	  $(INSTALL) $(THEME_DIR)/$$i $(DESTDIR)$(datadir)/icons/sugar/cursors/$$i ;	\
+	  $(INSTALL) -m u=rw,go=r $(THEME_DIR)/$$i $(DESTDIR)$(datadir)/icons/sugar/cursors/$$i ;	\
 	done
 
 uninstall-local:
