diff --git a/gtk/engine/sugar-style.c b/gtk/engine/sugar-style.c
index 0ffaf06..e85302d 100644
--- a/gtk/engine/sugar-style.c
+++ b/gtk/engine/sugar-style.c
@@ -662,8 +662,11 @@ sugar_style_draw_arrow (GtkStyle       *style,
     SugarInfo *info = &arrow_info.info;
     cairo_t *cr;
 
+// GTK_ARROW_NONE was implemented around GKT+ 2.10
+#ifdef GTK_ARROW_NONE
     if (arrow_type == GTK_ARROW_NONE)
         return;
+#endif
 
     if (width < 0 || height < 0)
         return;
