--- sugar-artwork-0.79.0.orig/gtk/theme/Makefile.am
+++ sugar-artwork-0.79.0/gtk/theme/Makefile.am
@@ -1,9 +1,9 @@
 sugar.gtkrc: gtkrc.em
-	$(srcdir)/em.py -p $$ -D theme=\'sugar\' $(srcdir)/gtkrc.em > \
+	empy -p $$ -D theme=\'sugar\' $(srcdir)/gtkrc.em > \
 		$(top_builddir)/gtk/theme/sugar.gtkrc
 
 sugar-xo.gtkrc: gtkrc.em
-	$(srcdir)/em.py -p $$ -D theme=\'sugar-xo\' $(srcdir)/gtkrc.em > \
+	empy -p $$ -D theme=\'sugar-xo\' $(srcdir)/gtkrc.em > \
 		$(top_builddir)/gtk/theme/sugar-xo.gtkrc
 
 clean:
