diff --git a/gtk/theme/Makefile.in b/gtk/theme/Makefile.in
index f229d6d..caee188 100644
--- a/gtk/theme/Makefile.in
+++ b/gtk/theme/Makefile.in
@@ -325,11 +325,11 @@ uninstall-am: uninstall-info-am uninstall-local
 	uninstall uninstall-am uninstall-info-am uninstall-local
 
 sugar-72.gtkrc: gtkrc.em
-	$(srcdir)/em.py -p $$ -D scaling=\'72\' $(srcdir)/gtkrc.em > \
+	empy -p $$ -D scaling=\'72\' $(srcdir)/gtkrc.em > \
 		$(top_builddir)/gtk/theme/sugar-72.gtkrc
 
 sugar-100.gtkrc: gtkrc.em
-	$(srcdir)/em.py -p $$ -D scaling=\'100\' $(srcdir)/gtkrc.em > \
+	empy -p $$ -D scaling=\'100\' $(srcdir)/gtkrc.em > \
 		$(top_builddir)/gtk/theme/sugar-100.gtkrc
 
 clean:
