diff --git a/icons/scalable/device/Makefile.in b/icons/scalable/device/Makefile.in
index 6cf306f..17bb98b 100644
--- a/icons/scalable/device/Makefile.in
+++ b/icons/scalable/device/Makefile.in
@@ -192,7 +192,6 @@ icon_DATA = \
 	battery-090.icon		\
 	battery-090.svg			\
 	battery-100.icon		\
-	battery-100.svg			\
 	battery-charging-100.svg	\
 	battery-charging-000.svg	\
 	battery-charging-010.svg	\
