--- a/gtk/engine/sugar-style.h
+++ b/gtk/engine/sugar-style.h
@@ -44,6 +44,11 @@
   GtkStyleClass parent_class;
 };
 
-G_GNUC_INTERNAL void sugar_style_register_type (GTypeModule *module);
+G_GNUC_INTERNAL 
+void         sugar_style_register_type  (GTypeModule *module);
+
+void         theme_init                 (GTypeModule *module);
+void         theme_exit                 (void);
+GtkRcStyle * theme_create_rc_style      (void);
 
 #endif /* __SUGAR_STYLE_H */
